docs/sources/tempo/traceql/metrics-queries.md

```diff
@@ -94,25 +97,37 @@ down by HTTP route.
 This might let you determine that `/api/sad` had a higher rate of erroring
 spans than `/api/happy`, for example.
 
-### The `count_over_time` and `min_over_time` functions
+### The `count_over_time`, `min_over_time` and `max_over_time` functions
 
 The `count_over_time()` let you counts the number of matching spans per time interval.
 
 ```
 { name = "GET /:endpoint" } | count_over_time() by (span.http.status_code)
 
 ```
+
 The `min_over_time()` let you aggregate numerical values by computing the minimum value of them, such as the all important span duration.
 
 ```
-{ name = "GET /:endpoint" } | min_over_time(duration) by (span.http.target)
+{ name = "GET /:endpoint" } | min_over_time(duration` by (span.http.target)
 ```
 
 Any numerical attribute on the span is fair game.
 
 ```
 { name = "GET /:endpoint" } | min_over_time(span.http.status_code)
 ```
+
+The `max_over_time()` let you aggregate numerical values by computing the maximum value of them, such as the all important span duration.
+
+```
+{ name = "GET /:endpoint" } | max_over_time(duration` by (span.http.target)
```

Comment:
Same question here. It seems like the duration' should be duration), based on the other examples.